vm/dependencies.go

```diff
@@ -51,6 +51,7 @@ type Config struct {
 	ProcessingBuildSkip              int             `json:"processingBuildSkip"`
 	TargetGossipDuration             time.Duration   `json:"targetGossipDuration"`
 	BlockCompactionFrequency         int             `json:"blockCompactionFrequency"`
+	EnableStateReadRPCHandler        bool            `json:"enableStateReadRPCHandler"`
```

Comment:
nit: Make this config key name consistent w/ the API (i.e not `StateRead`)

Reply:
I renamed it as `EnableJSONRPCStateHandler`

It follows the mapping I found in the code base
"JSONRPCEndpoint" -> "jsonRPCHandler" -> "NewJSONRPCServer".
So it gives in the PR:
"JSONRPCStateEndpoint" -> "EnableJSONRPCStateHandler" -> "NewJSONRPCStateServer"